backend/tracim_backend/applications/agenda/schemas.py

```diff
@@ -36,7 +41,9 @@ class AgendaFilterQuerySchema(marshmallow.Schema):
     workspace_ids = StrippedString(
         validate=regex_string_as_list_of_int,
         example="1,5",
-        description="comma separated list of included workspace ids",
+        description="comma separated list of included workspace ids,"
+        "setting this parameters will disable"
```

Comment:
There is some missing comas here

Reply:
Probably not (https://stackoverflow.com/questions/34174539/python-string-literal-concatenation), but it sure lacks a space after the comma on line 44 :-)

Reply:
fixed